cluster/ci/monitoring/prometheus_rule_prow.yaml

```diff
@@ -50,7 +50,7 @@ spec:
           and <https://prow.svc.ci.openshift.org/?job=release-*-upgrade*|deck-portal>.'
       expr: |
         sum(rate(prowjob_state_transitions{job="plank",job_name=~"release-.*-upgrade.*",job_name!~"rehearse.*",state="success"}[30m]))/sum(rate(prowjob_state_transitions{job="plank",job_name=~"release-.*-upgrade.*",job_name!~"rehearse.*",state=~"success|failure"}[30m])) * 100 < 100
```

Comment:
Let's change the vector expression `[30m]`, not the alert duration

Reply:
my bad. done.